### PR TITLE
Easy install script 

### PR DESCRIPTION
**Pull request by kamikazechaser**
Easy install script for debian based distros

Tested to be working on Ubuntu 16.04

### CODE REVIEW

**Comment by Benjarobbi** [comment]
you did forget to chmod the file before running ^^

**Comment by kamikazechaser** [comment]
@Benjarobbi Did you check out how the script works in the [README](https://github.com/RememberTheAir/GroupButler/pull/136/files#diff-04c6e90faac2675aa89e2176d2eec7d8R36)?

**Comment by Benjarobbi** [comment]
oh sorry. my fault :sweat_smile:

**Comment by kamikazechaser** [comment]
@Benjarobbi No worries 😃

**Comment by mymedia2** [comment]
I think you should have used Luarocks for solving the problem. See [here](https://github.com/luarocks/luarocks/wiki/Creating-a-rock).
Also why do you download installation script without certificate validation? 
Besides you run `sudo su`, but then your script executes `sudo` before each command. It seems redundant.

**Comment by kamikazechaser** [comment]
@mymedia2 1. Umm I dont understand what you mean by creating a lua rock package, this is basically just an installation script which saves time, nothing to do with lua
2.the wget is running with no certificate validation because some servers may be denied access to GitHub for reasons better known to GitHub. However this does nit affect anything as its just pulling the install file from GitHub, no biggie there, refer to wget documentation
3. The initial sudo su is to run the script without chomding it (you can alternatively chomd it and run without sudo su) I find this method shorter and less complicated 😄

**Comment by mymedia2** [comment]
> some servers may be denied access to GitHub
If someone faces with this issue, he should solve it by other way. Otherwise, HTTPS encryption doesn't make any sense. ISP can intercept the request and arrange MitM attack. They can palm off an arbitrary code that will be executed with superuser privileges! Today GitHub has the valid certificate, and there are no significant reasons to ignore security. 

> what you mean by creating a lua rock package
I thought it would be useful to specify all Lua dependencies and their versions in special `*.rockspec` file. 

By the way there is no need to give anyone rights to modify the `launch.sh` file. This is unsafe. Git will automatically make appropriate access permissions to this file during cloning.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    sudo luarocks install redis-lua
+    sudo luarocks install lua-term
+    sudo luarocks install serpent
+    sudo luarocks install dkjson
+    sudo luarocks install lanes
+    sudo luarocks install Lua-cURL
+    
+    echo -e "\e[1;36mFetching latest Group Butler source code\e[0m"
+    git clone https://github.com/RememberTheAir/GroupButler.git && cd GroupButler
+    sudo chmod 777 launch.sh
```
Comment (mymedia2) [inline install.sh]: Yes, it seems we can remove this line without any consequences

Comment (mymedia2) [inline install.sh]: Also I think it has to be removed from README

**Comment by kamikazechaser** [comment]
I have updated it

**Comment by kamikazechaser** [comment]
@mymedia2 Something like a package.json file for lua instead?

**Comment by kamikazechaser** [comment]
@mymedia2 https://github.com/luarocks/luarocks/issues/573

**Comment by mymedia2** [comment]
It looks much better +1

I agree with @hishamhm in that thread to which you referenced. The idea to write a rockspec with dependencies looks very good. 

> Something like a package.json file for lua instead? 

Yes, scripts for NodeJS have something similar

**Pull request merged**